jira/github_jira.py

```diff
@@ -77,7 +88,7 @@
     issue_template = {
         'project': 'ACA',
         'summary': '[{0}/{1}] {2}'.format(bug.repository.name, bug.number, bug.title),
-        'description': '{0} \n {1}'.format(bug.html_url, bug.body),
+        'description': '{0} \n {1} \n {2} \n "-------------" \n {3}'.format(bug.html_url, bug.body, dod_title, dod_content),
```

Comment:
```suggestion
        'description': '{0} \n {1} \n h3. {2} \n {3}'.format(bug.html_url, bug.body, dod_title, dod_content),
```

I think this should work and keep the formatting consistent with how other tickets are formatted.